weaver/wps_restapi/swagger_definitions.py

```diff
@@ -228,6 +228,19 @@ class URL(ExtendedSchemaNode):
     format = "url"
 
 
+class DATETIME_INTERVAL(ExtendedSchemaNode):
+    schema_type = String
+    description = "DateTime name pattern."
```

Comment:
Provide a more specific `description` and an `example` value because it is not clear how to use the field when looking at the generated OpenAPI.